overlord/overlord.go

```diff
@@ -163,6 +165,18 @@ func New() (*Overlord, error) {
 	return o, nil
 }
 
+func (o *Overlord) EnableHotplug() error {
+	udevMon := hotplug.CreateUDevMonitor(o.ifaceMgr.HotplugDeviceAdded, o.ifaceMgr.HotplugDeviceRemoved)
+	if err := udevMon.Connect(); err != nil {
+		return err
+	}
+	if err := udevMon.Run(); err != nil {
```

Comment:
Do you need to disconnect in case Connect works but Run fails?

Reply:
Right. Not strictly needed (not critical) since we exit immediately afterwards if EnableHotplug fails, nonetheless it's a good idea to keep it clean. Added Disconnect method.

Reply:
Note that `Run()` as it currently defined always returns `nil`. Maybe that's changed in subsequent PRs.